### PR TITLE
Upgrade Erlang OTP from 19.3.6.4 to 20.3.5

### PR DESCRIPTION
**Pull request by kitsirota**
Hello,
We have had an Erlang process leak in Erlang 19 which has caused cluster instability and performance issues for at least a few months.  We updated Erlang OTP in place and have been running this dev build in pre-prod for about a week.  Since the upgrade, we have no observed any leaks.    

Changes in this PR:
1. Updated sha for golang in blob.yml.  It looks like the sha in the repo does not match the golang asset in the blobstore.  I've updated it with the sha1 reported by the cli, but it would be great to get a confirmation on which sha should be there.  If I missed something on my end, this change can be omitted from the PR all together.
2. Erlang package has been updated in place from 19.3.6.4 to 20.3.5 and rabbitmq-server jobs were updated accordingly.  Autoconf package was added as a build dependency for erlang.
-The erlang package was ported from https://github.com/rabbitmq/rabbitmq-server-boshrelease/tree/master/packages/erlang-20.3.5.  
-The update erlang package requires autoconf to build, this was also ported from https://github.com/rabbitmq/rabbitmq-server-boshrelease/tree/master/packages/autoconf-2.69.  

Package Sources:
-Autoconf 2.69 - https://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.xz
-Erlang OTP - https://github.com/erlang/otp/releases/tag/OTP-20.3.5

Open Questions:
1. Do we want to rollout OTP 20 as an in-place package bump or in the style of rabbitmq-server.version?
2. To build and test this PR I was dropping the tarballs into the src directory.  Is the list of sources above sufficient to add these as blobs to this project's S3 bucket?  

Thank you!
Kit

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@kitsirota Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=pivotal-cf/cf-rabbitmq-release&pullRequestId=81)!

**Comment by kitsirota** [comment]
Hi folks, Im psyched to see the this PR was "scheduled."  Please let me know if you have any questions or suggestions on the approach.  Im also available on CloudFoundry slack under @kit.sirota

Thanks!

**Comment by blgm** [comment]
Hi @kitsirota,

TL;DR we have just released [v245.0.0](https://github.com/pivotal-cf/cf-rabbitmq-release/releases/tag/v245.0.0) which upgrades to Erlang 20.3.5

Thank you for your pull request.  We have used it as inspiration rather than merging it for the following reasons:
- we wanted to create automation to do this for us in future - and have now done so
- we needed to filter out some files for open source licensing reasons
- we run autoconf on Erlang prior to creating the tar file (which becomes the blob).  This is for historical reasons, and maybe should be changed, but we did not want to make that change at the same time as an upgrade

Thank you for your contribution.  We would welcome feedback on your experience of contributing, and how the latest release works for you.
